engine/source/materials/mat_share/mulaw.F90

```diff
@@ -633,6 +634,13 @@ subroutine mulaw(timers,&
             sv5(i) = zero
             sv6(i) = zero
             e7(i)  = zero
+            !
+            svo1(i) = zero
+            svo2(i) = zero
+            svo3(i) = zero
+            svo4(i) = zero
+            svo5(i) = zero
+            svo6(i) = zero
```

Comment:
Hi @SaidMamouri,
Are we sure that we need this initialization?
If I understand well, this increases the cost of every material law, but is necessary only if ivisc == 3 ?